src/.vuepress/config.js

```diff
@@ -84,11 +72,11 @@ module.exports = {
                 items: [
                   {
                     text: 'Migrate from 0.x',
-                    link: '/miscellaneous/migration-guide.html'
+                    link: 'https://github.com/innovolve-ai/typerefinery'
                   },
                   {
                     text: 'Changelog',
-                    link: 'https://github.com/vuejs/vuepress/blob/master/CHANGELOG.md'
+                    link: 'https://github.com/innovolve-ai/typerefinery/releases'
```

Comment:
I believe this URL should be not "github.com", it should be the application Domain's respective URL

Reply:
Maxx asked to keep this link for the meantime, it will be changed afterwards.